src/containers/teleport/pages/Layout/Layout.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import { Outlet, useLocation } from 'react-router-dom';
+import s from './Layout.module.scss';
+import TabList from '../../components/tabList/TabList';
+import { TypePages } from '../../type';
+
+function Layout() {
+  const location = useLocation();
+  const [active, setActive] = useState<TypePages | undefined>(TypePages.swap);
+
+  useEffect(() => {
+    const locationSplit = location.pathname.replace(/^\/|\/$/g, '').split('/');
+    setActive(
+      Object.values(TypePages).find((item) => item === locationSplit[1])
+    );
+  }, [location]);
```

Comment:
why here need hook and state? possible to write only in render, and maybe `useMemo` if need